src/Core/src/Platform/Windows/WindowRootView.cs

```diff
@@ -122,6 +123,11 @@ internal void UpdateAppTitleBar(int appTitleBarHeight, bool useCustomAppTitleBar
 				WindowTitleBarContentControlVisibility = UI.Xaml.Visibility.Collapsed;
 			}
 
+			var flowDirectionSetting = ResourceContext.GetForViewIndependentUse().QualifierValues["LayoutDirection"];
+			WindowTitleBarFlowDirection = flowDirectionSetting == "LTR" ?
+				UI.Xaml.FlowDirection.LeftToRight : UI.Xaml.FlowDirection.RightToLeft;
```

Comment:
Should this default to `LeftToRight` if the `flowDirectionSetting` is somehow blank one day? That would be a weird bug:
```suggestion
			WindowTitleBarFlowDirection = flowDirectionSetting == "RTL" ?
				UI.Xaml.FlowDirection.RightToLeft : UI.Xaml.FlowDirection.LeftToRight;
```
_NOTE: I don't know if it will say `"RTL"`, I don't know this API_